ratex/optimizer/adam.py

```diff
@@ -118,22 +121,29 @@ def step(self, closure=None):
                         # state['exp_avg_sq'] = torch.zeros_like(p, memory_format=torch.preserve_format)
                         # pylint: enable=line-too-long
                         if self._need_partition(param):
-                            state["exp_avg"] = self._create_partitioned_buffer(
-                                param, dtype=torch.float32
+                            state["exp_avg"] = _RATEXC._raf_mark_parameter(
```

Comment:
Are states in pt native adam marked?

Reply:
not sure but I think it might not get marked correctly either